UKHomeOffice/nmsw-ui
# NMSW-421 Change L headings to XL

# Ticket

NMSW-421

----

## AC

Change heading on cookies, privacy and accessibility pages from l to xl

----

## To test

- Check that all regulatory pages have a class of `govuk-heading-xl`
- Run the app
- All the footer pages should have an xl heading

----

## Notes
